lib/jira/commands/install.rb

```diff
@@ -15,35 +15,52 @@ class Install < Base
 
       def run
         io.say('Please enter your JIRA information.')
-        inifile[:global] = params
+        inifile[:global] = base_params
+        inifile.write # Do this now because cookie authentication uses api calls
+
+        inifile.delete_section("cookie") if inifile.has_section?("cookie")
+        case authentication
+        when "basic"
+          inifile[:global][:password] = password
+        when "token"
+          inifile[:global][:token] = token
+        when "cookie"
+          response = cookie
+          inifile[:cookie] = {}
+          inifile[:cookie][:name] = response['name']
+          inifile[:cookie][:value] = response['value']
+        else
+          io.error("A valid authentication type was not selected.")
```

Comment:
this only way we could get into this state is if the user aborted the selection prompt, so we can probably get rid of the `else logic`